test/test.js

```diff
@@ -3754,6 +3754,67 @@ var testFixture = {
             }
         },
 
+        'var o = {\n/** Desc */\nfoo: function(){}\n};': {
```

Comment:
It's unfortunate that this file is not nearly a JSON file. We are currently using a JavaScript style here and it normally uses single quotes, without quoting the keys, and curly braces are on the next line. You can easily pick up the style from the tests around you.